using-babelfish/uast-querying.md

```diff
@@ -4,53 +4,41 @@ The clients support querying over the returned UAST to filter for specific nodes
 
 The querying uses [xpath syntax](https://www.w3.org/TR/xpath/) and it can be really useful to extract features from the code and, thanks to the annotation system implemented by babelfish, it can be done in a universal way.
 
-Any of the [node](https://godoc.org/gopkg.in/bblfsh/sdk.v2/uast#Node) fields can be used for querying in the following way:
+Any of the [Node](https://godoc.org/gopkg.in/bblfsh/sdk.v2/uast#Node) fields can be used for querying in the following way:
 
-* `InternalType` is converted to the element name
-* `Token`, if available, is converted to an attribute with `token` as keyword and the actual token as value
-* Every `Role` is converted to an attribute concatenating a `role` prefix and the role name in CamelCase.
+* `@type` is converted to the element name
+* `@token`, if available, is converted to an attribute with `@token` as keyword and the actual token as value
+* Every `Role` is converted to an attribute with `@role` as keyword and the role name as value.
 * Every `Property` is converted to an attribute with the property keyword as keyword and the property value as value
-* `StartPosition`, if available, is mapped to three attributes:
-  * A `startOffset` attribute, with the offset as value
-  * A `startLine` attribute, with the line as value
-  * A `startCol` attribute, with the column as value
-* `EndPosition`, if available, is mapped to three attributes:
-  * A `endOffset` attribute, with the offset as value
-  * A `endLine` attribute, with the line as value
-  * A `endCol` attribute, with the column as value
+* `@pos`, if available, is mapped to a node with a `start` and `end` properties, each containing a node of `Position` type which in turn will have the `offset`, `line` and `col` properties.
 
 Internally, these are mapped in the XML node in the following way:
 
 ```markup
-<{{InternalType}}
-    token='{{Token}}'
+<{{@type}}
+    @token='{{Token}}'
     {{for role in Roles}}
-    role{{role}}
+    @role{{role}}
     {{for key, value in Properties}}
     {{key}}='{{value}}'
-    startOffset={{StartPosition.Offset}}
-    startLine={{StartPosition.Line}}
-    startCol={{StartPosition.Col}}
-    endOffset={{EndPosition.Offset}}
-    endLine={{EndPosition.Line}}
-    endCol={{EndPosition.Col}}>
+    @pos{{start, end}}
     {{Children}}
 </{{InternalType}}>
```

Comment:
`s/InternalType/@type/` ?

Otherwise 🤔 
<img width="320" alt="screen shot 2018-10-15 at 11 01 11 am" src="https://user-images.githubusercontent.com/5582506/46937350-bca6ed80-d069-11e8-9aab-b76c84b4a45a.png">

Reply:
Fixed, thanks!